services/dumper.js

```diff
@@ -1,4 +1,5 @@
 'use strict';
+const { getDatabaseUrl } = require('utils');
```

Comment:
As a convention, we try to require our own components after external dependencies (bluebird, fs, lodash,... ).
I feel it makes the code more readable.